qutebrowser/browser/browsertab.py

```diff
@@ -895,3 +895,7 @@ def __repr__(self):
 
     def is_deleted(self):
         return sip.isdeleted(self._widget)
+
+    def should_show_icon(self):
```

Comment:
I try to avoid having "utility" stuff (which likely wouldn't be an useful API for future plugins) on the `BrowserTab` object. What about moving this to `TabData` as it's closely related to that anyways?